src/modules/utils.js

```diff
@@ -280,6 +280,14 @@ self = module.exports = {
             !url.href.match(/^https?:\/\/|^file:\/\//);
 
         return !!sameOrigin;
+    },
+
+    isNumber: function (value) {
+        if (typeof value === 'number') {
+            return true;
+        }
+
+        return !!value.match(/-?(\d+|\d+\.\d+|\.\d+)([eE][-+]?\d+)?/);
```

Comment:
It depends on what we want to validate here. If we go with the `parseFloat` approach then `1.2hello` is a number equal to `1.2`. Similarly, with the current approach we forbid the european style `1,2`, but that would be incorrectly parsed as `1.0` if we allowed it.